python/kfserving/test/test_server.py

```diff
@@ -234,14 +244,32 @@ def app(self):  # pylint: disable=no-self-use
         server.register_model(model)
         return server.create_application()
 
-    async def test_predict_ce_binary(self, http_server_client):
+    async def test_predict_ce_binary_dict(self, http_server_client):
         event = dummy_cloud_event({"instances":[[1,2]]}, set_contenttype=True)
         headers, body = to_binary(event)
         resp = await http_server_client.fetch('/v1/models/TestModel:predict',
                                               method="POST",
                                               headers=headers,
                                               body=body)
+    
+        assert resp.code == 200
+        assert resp.body == b'{"predictions": [[1, 2]]}'
+        assert resp.headers['content-type'] == "application/x-www-form-urlencoded"
+        assert resp.headers['ce-specversion'] == "1.0"
+        assert resp.headers['ce-id'] == "36077800-0c23-4f38-a0b4-01f4369f670a"
+        assert resp.headers['ce-source'] == "https://example.com/event-producer"
+        assert resp.headers['ce-type'] == "com.example.sampletype1"
+        assert resp.headers['ce-datacontenttype'] == "application/x-www-form-urlencoded"
+        assert resp.headers['ce-time'] > "2021-01-28T21:04:43.144141+00:00"
 
+    async def test_predict_ce_binary_json(self, http_server_client):
+        event = dummy_cloud_event(json.loads('{"instances":[[1,2]]}'), set_contenttype=True)
```

Comment:
this test looks identical to the above one, `json.loads('{"instances":[[1,2]]}')` effectively is the same as `{"instances":[[1,2]]}`

Reply:
Ya that's true. 
I think I'll just remove the test. 
I tried to add this to replicate the json CE issue.